lib/pbench/test/unit/server/test_endpoint_configure.py

```diff
@@ -52,6 +52,7 @@ def check_config(self, client, server_config, host, my_headers={}):
                 "datasets_list": f"{uri}/datasets/list",
                 "datasets_mappings": f"{uri}/datasets/mappings/",
                 "datasets_metadata": f"{uri}/datasets/metadata/",
+                "datasets_inventory_": f"{uri}/datasets/inventory//",
```

Comment:
Siddardh; you need to rebase on `main` again, and you'll need to update this because the trailing `_` in the key and the trailing url `/` characters are gone now. As written, this test will fail once you merge/rebase.